charts/block-node-server/values-overrides/backfill.yaml

```diff
@@ -2,6 +2,12 @@ blockNode:
   config:
     BACKFILL_BLOCK_NODE_SOURCES_PATH: "/opt/hiero/block-node/backfill/block-node-sources.json"
     BACKFILL_FETCH_BATCH_SIZE: "10"
+    # Optional: gRPC client configuration overrides
```

Comment:
This is needed. Configs are already documented and this file is supposed to contain the minimum override to enable backfill